Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG ATLANTIS
-FROM runatlantis/atlantis:v${ATLANTIS}
+FROM ghcr.io/runatlantis/atlantis:v${ATLANTIS}
```

Comment:
M$ exodus :)

Reply:
It is forced by atlantis repository. Mentioned here: https://github.com/runatlantis/atlantis/releases/tag/v0.18.5